src/dso_api/dynamic_api/serializers.py

```diff
@@ -232,7 +232,11 @@ def _loose_relation_m2m_fields(self):
                     to_snake_case(part) for part in field.relation.split(":")
                 ]
                 url_name = f"{app_name}:{dataset_name}-{table_name}-detail"
-                result.append((f.attname, toCamelCase(f.attname), url_name))
+                related_ds = field.related_model.get_dataset()
+                related_identifier_field = getattr(related_ds, "identifier")
```

Comment:
Dit is identiek aan: `related_identifier_field = related_ds.identifier`.
Tenzij je `getattr(related_ds, "identifier", None)` bedoeld.

Reply:
De getattr is hier idd overbodig.